drip/utils.py

```diff
@@ -59,16 +61,16 @@ def get_full_field(parent_field: str, field_name: str) -> str:
     return full_field
 
 
-def get_rel_model(field, RelatedObject):
-    if isinstance(field, RelatedObject):
+def get_rel_model(field: FieldType, RelatedObject: RelatedObject) -> Type[models.Model]:
+    if isinstance(field, RelatedObject):  # type: ignore
```

Comment:
Why do you need the ignore here?

Reply:
It raise an error related to typing in the function `isinstance`, I want to apply something only in RelatedObject type. But Mypy doesn't get it that way. The only workaround I find is to ignore it.